packages/codemods/src/variantColor-to-colorScheme.ts

```diff
@@ -0,0 +1,128 @@
+import { Transform } from "jscodeshift"
+import { describe } from "jscodeshift-helper"
+import { types as t } from "@babel/core"
+
+const renames = [
+  ["variantColor", "colorScheme"],
+  // ["size", "boxSize"], // need to break out into w and h props in the future
+  ["rounded", "borderRadius"],
+  ["roundedLeft", "borderLeftRadius"],
+  ["roundedRight", "borderRightRadius"],
+  ["roundedTop", "borderTopRadius"],
+  ["roundedTopLeft", "borderTopLeftRadius"],
+  ["roundedTopRight", "borderTopRightRadius"],
+  ["roundedBottomLeft", "borderBottomLeftRadius"],
+  ["roundedBottomRight", "borderBottomRightRadius"],
+  ["roundedBottom", "borderBottomRadius"],
+  ["size", "boxSize"], // do we need to change all size to boxSize or just for Box component
```

Comment:
We need to careful with this one. Some components use the `size` prop to refer to visual size and not width+height. Just like button, checkbox, etc.

For these components, we won't change the prop. but we'll update for the rest. I guess the best solution is to keep an array of the components to ignore.

Reply:
Yeah we're definitely going to use `width`+`height`. Unfortunately `jscodeshift` is kind of a pain, so we very quickly got stuck but we're going to keep working on it and see where we can get.

Reply:
Oh, my comment only addressed half of your point lol. We're planning on changing the prop name only if the value of `size` isn't one of the existing t-shirt sizes we use in `master`.